variables.tf

```diff
@@ -174,3 +174,6 @@ variable "cfssl_data_volumeid" {}
 variable "etcd_data_volumeids" {
   type = "list"
 }
+
+variable "account" {}
```

Comment:
What is account in this instance?

Reply:
dev / prod. It is needed because bucket names are global

Reply:
so it could also be environment? `${var.environment}`